### ISSUE TITLE
aborting due to: PyEval_InitThreads’ is deprecated

### ISSUE THREAD

**Issue by ThomasWaldmann** [python-llfuse/python-llfuse#27]
doesn't compile on travis-ci, ubuntu focal / python 3.9-dev:
```
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -fPIC -I/home/travis/virtualenv/python3.9-dev/include -I/opt/python/3.9-dev/include/python3.9 -c src/llfuse.c -o build/temp.linux-x86_64-3.9/src/llfuse.o -D_FILE_OFFSET_BITS=64 -I/usr/include/fuse -DFUSE_USE_VERSION=29 -Wall -Wextra -Wconversion -Wsign-compare -DLLFUSE_VERSION="1.3.7" -Wno-unused-function -Wno-implicit-fallthrough -Wno-unused-parameter -Werror -Wfatal-errors -Wno-error=unreachable-code -Werror=conversion -Werror=sign-compare

src/llfuse.c: In function ‘__pyx_f_6llfuse_session_loop_mt’:
src/llfuse.c:42419:3: error: ‘PyEval_InitThreads’ is deprecated [-Werror=deprecated-declarations]
42419 |   PyEval_InitThreads();
      |   ^~~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
cc1: all warnings being treated as errors
```

**Comment by ThomasWaldmann** [comment]
https://travis-ci.org/github/python-llfuse/python-llfuse/jobs/732700895

**Comment by ThomasWaldmann** [comment]
related to #22 ?

**Comment by ThomasWaldmann** [comment]
this was fixed for travis-ci testing by not enabling DEVELOPER_MODE on travis-ci, see #26.

**Comment by ThomasWaldmann** [comment]
the aborting is fixed by #30.
